personio-personnel-data-api-oa3.yaml

```diff
@@ -2216,6 +2216,128 @@ paths:
                           "The file must be a file of type: pdf, docx, doc, jpg, zip, png, txt, jpeg, odt, xlsx, rtf, htm, xls, p7s, pptx, pages, rar, ppt, gif, tif, html, msg, asc, tiff, wps, bmp, 7z, csv, ics, vcf, dotx, numbers, eps, gz, ods, otf, odp, odg, rtfd, txz, webarchive, xml, xps, md, fax."
                         ]
 
+  /company/custom-reports/reports:
+    get:
+      tags:
+        - Custom Reports
+      parameters:
+        - name: X-Company-Id
+          in: header
+          schema:
+            type: integer
+          required: true
+        - name: report_ids
+          in: query
+          schema:
+            type: array
+            items:
+              type: string
+          required: false
+        - name: status
+          in: query
+          schema:
+            type: string
+          example: "up_to_date"
+          required: false
+      summary: List of reports
+      operationId: listReports
+      responses:
+        200:
+          description: Ok
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/PublicListReportsResponse"
+        500:
+          description: Internal Server Error
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/PublicAPIException"
+
+  /company/custom-reports/reports/{reportId}:
```

Comment:
[Suggestion]: In the previous GET request, the list of report IDs was provided as `report_ids`, here we have `reportId`. Could we maybe have `report_id` here for consistency reasons?